test/extended/operators/resources.go

```diff
@@ -104,9 +137,7 @@ var _ = g.Describe("[sig-arch] Managed cluster", func() {
 						}
 					}
 				case "Job":
-					if pod.Namespace == "openshift-marketplace" {
-						ref.Name = "<batch_job>"
-					}
```

Comment:
before, this would have set `ref.Name` only for jobs in `openshift-marketplace`.
after, it will set it for jobs in any namespace (except the exempt ones, so *not* in `openshift-marketplace`).
just wanted to make sure that's the logic change you wanted. otherwise, lgtm

Reply:
Yeah it was intentional, if we're excluding a job we need to replace the Name.  In a previous iteration I was excluding specific ROSA jobs, like this:

```
"batch/v1/Job/openshift-sre-pruning/<batch_job>/container/deployments-pruner/limit[memory]":                                                                   "https://issues.redhat.com/browse/OSD-21708",
```

but it wasn't working.  I ended up deciding to just exclude the entire namespace since I had so many exceptions, but I left this so excluding jobs works in case someone adds one again in the future. There's no reason to gate this on the namespace